tests/test_integration.py

```diff
@@ -26,10 +27,38 @@ def test_get_genotype_reactions():
     assert set(result.keys()) == {'BBa_J23100', 'AB326105', 'NP_600058', 'BBa_0010'}
 
 
-def test_reactions_additions():
-    ecoli_original = storage.get('iJO1366').model.copy()
-    ecoli = ecoli_original.copy()
-    ecoli.notes['changes'] = get_empty_changes()
+def test_reactions_additions(monkeypatch, iJO1366):
```

Comment:
I'm okay with leaving this test case as is for now but in future it should be broken up because it tests too many pieces of logic in one.